ci/conda_env_python.yml

```diff
@@ -22,6 +22,7 @@ numpy>=1.14
 pandas
 pytest
 pytest-faulthandler
+pytest-lazy-fixture
```

Comment:
Is it still required? It doesn't seem so.

Reply:
Required, see my reference below.

Reply:
purest has an rfc about including this functionality in the core.